src/tray/templates/services/app/deploy.yml.j2

```diff
@@ -1,25 +1,27 @@
+{% for peertube_runner in peertube_runners %}
+---
 apiVersion: v1
 kind: Deployment
 metadata:
   labels:
     app: peertuberunner
-    service: app
+    service: "app-{{ peertube_runner.name | replace('_', '-')}}"
     version: "{{ peertube_runner_image_tag }}"
-  name: "peertuberunner-app"
+  name: "peertuberunner-app-{{ peertube_runner.name | replace('_', '-')}}"
```

Comment:
Just for brevity, can we use a variable here ?  something like

    {% set runner_dashed_name = peertube_runner.name | replace('_', '-') %}

Reply:
Great idea